packages/nodes-base/nodes/Airtop/actions/session/create.operation.ts

```diff
@@ -63,24 +69,57 @@ export const description: INodeProperties[] = [
 				description: 'Use Airtop-provided proxy',
 			},
 			{
-				name: 'Custom',
-				value: 'custom',
-				description: 'Configure a custom proxy',
+				name: 'Proxy URL',
+				value: 'proxyUrl',
+				description: 'Use a proxy URL to configure the proxy',
 			},
 		],
 		displayOptions,
 	},
+	{
+		displayName: 'Proxy Configuration',
+		name: 'proxyConfig',
+		type: 'collection',
+		default: { country: 'US', sticky: true },
+		description: 'The Airtop-provided configuration to use for the proxy',
+		placeholder: 'Add Attribute',
+		options: [
+			{
+				displayName: 'Country',
+				name: 'country',
+				type: 'options',
+				default: 'US',
+				description:
+					'The country to use for the proxy. Not all countries are guaranteed to provide a proxy. Learn more <a href="https://docs.airtop.ai/api-reference/airtop-api/sessions/create#request.body.configuration.proxy.Proxy.Airtop-Proxy-Configuration.country" target="_blank">here</a>.',
+				options: countryOptions,
+			},
+			{
+				displayName: 'Keep Same IP',
+				name: 'sticky',
+				type: 'boolean',
+				default: true,
+				description:
+					'Whether to try to maintain the same IP address for the duration of the session. Airtop cannot guarantee that the same IP address will be available for 30 minutes at a time.',
```

Comment:
This sentence is a bit confusing. Maybe the following? The phrasing on the docs is confusing so I think this is what is meant. https://docs.airtop.ai/api-reference/airtop-api/sessions/create#request.body.configuration.proxy.Proxy.Airtop-Proxy-Configuration.sticky.
@danielsht86 is this right?
```suggestion
					'Whether to try to maintain the same IP address for the duration of the session. Airtop can guarantee that the same IP address will be available for up to a maximum of 30 minutes.',
```